### ISSUE TITLE
USER ISSUE: tree chopping crash

### ISSUE THREAD

**Issue by Lazersgm** [StrangeLoopGames/EcoIssues#6952]
**Version:** 0.7.3.1 beta

**Steps to Reproduce:**
just switched from fishing pole to axe and hit a tree i choped down before i exited and saved the game. not sure what happened

**Expected behavior:**

**Actual behavior:**
The following error message popped up:
```
Caught exception

<size=60.00%>Object reference not set to an instance of an object.

   at TreeEntity.TrySliceTrunk(Player player, Single slicePoint)
   at Eco.Gameplay.Items.WeaponItem.OnActLeft(InteractionContext context)
   at Eco.Mods.TechTree.AxeItem.OnActLeft(InteractionContext context)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)
</size>
```

**Comment by darkshad9999** [comment]
Update to the latest version this was fixed on 0.7.3.2 Beta 

This issue has been reported by multiple people and the devs have
said they have fixed it in 0.7.3.2 which is released.

https://github.com/StrangeLoopGames/EcoIssues/issues/6830
https://github.com/StrangeLoopGames/EcoIssues/issues/6850
https://github.com/StrangeLoopGames/EcoIssues/issues/6866
https://github.com/StrangeLoopGames/EcoIssues/issues/6915
https://github.com/StrangeLoopGames/EcoIssues/issues/6915
https://github.com/StrangeLoopGames/EcoIssues/issues/6927
https://github.com/StrangeLoopGames/EcoIssues/issues/6944

**Comment by MarjieAVolk** [comment]
Thanks, fixed in 7.3.2.

**Comment by darkshad9999** [comment]
Ya and this release isn't on Steam yet and the server I am playing on this guy says he can only 
update the game though steam.  When is this patch going to be on steam so people that rely on steam can update it. The server I am playing on the admin says that he can only update though steam right now you have 0.7.3.2 on your website but NOT on steam!  Please get 0.7.3.2 on steam!  So the people that use steam can update their servers too!
